kmk/extensions/peg_oledDisplay.py

```diff
@@ -0,0 +1,133 @@
+import busio
+import gc
+
+import adafruit_displayio_ssd1306
+import displayio
+import terminalio
+from adafruit_display_text import label
+
+from kmk.extensions import Extension
+
+
+class oled(Extension):
+    def __init__(
+        self,
+        board,
+        views,
+        toDisplay: str = 'TXT',
+        oWidth: int = 128,
+        oHeight: int = 32,
+        flip: bool = False,
+    ):
+        displayio.release_displays()
+        self._views = views
+        self._toDisplay = toDisplay
+        i2c = busio.I2C(board.SCL, board.SDA)
+        self.width = (oWidth,)
+        self.height = (oHeight,)

```

Comment:
since these are never used again (and tuples aren't negligible in memory), don't store these on `self` in tuple form. storing the int representation is fine, but the tuples should be passed to the SSD1306 constructor and then allowed to be garbage collected (or whatever SSD1306 does to them)